admin_scripts/os2borgerpc_kiosk/chromium_install.sh

```diff
@@ -1,6 +1,12 @@
 #!/bin/bash
 # Minimal install of X and Chromium and connectivity.
 
+set -e
+
+export DEBIAN_FRONTEND=noninteractive
+
+apt-get update -y
+
 apt-get install -y xinit xserver-xorg-core x11-xserver-utils --no-install-recommends --no-install-suggests
 apt-get install -y xdg-utils xserver-xorg-video-qxl xserver-xorg-video-intel xserver-xorg-video-all xserver-xorg-input-all libleveldb-dev
-apt-get install -y chromium-browser > /dev/null
+apt-get install -y chromium-browser
```

Comment:
Det er eller har i hvert fald været nødvendigt at have den redirect til `/dev/null`. Problemet er, at Chromium-installationen udskriver "scroll"-kommentarer for at have én linje, der bladrer forbi i stedet for at fylde skærmen, og dette giver invalid XML, når svaret sendes tilbage til serveren. Dette var den workaround jeg nåede frem til ...